README.md

```diff
@@ -10,6 +10,12 @@
 
 Apache HoraeDB (incubating) is a high-performance, distributed, cloud native time-series database.
 
+> [!IMPORTANT]
+> Apache HoraeDB (incubating) is an effort undergoing incubation at the Apache
+> Software Foundation (ASF), sponsored by the Apache Incubator PMC.
+>
+> Please read the [DISCLAIMER](DISCLAIMER) and a full explanation of ["incubating"](https://incubator.apache.org/policy/incubation.html).
```

Comment:
I can instead use the exact content of the DISCLAIMER. But since we're seeking for a shorter description, here is a proposal that I suppose is clear and concise.

cc @justinmclean @sebbasf

Reply:
> the linked version

I don't know what this is. Would you send a suggestion or patch to show it?

> but either would do

So the version proposed here is OK to you?

Reply:
@sebbASF Thanks for sharing your opinion. Could you provide a sentence that I can use here?

I'd prefer to try the "link to README.md" way but I may not make a precious sentence :P